docs/content/common-subs/required-packages.md

```diff
@@ -60,7 +60,141 @@
+        ##2.2 Select a linux distribution and install it into WSL, like this:
+        ```
+        wsl --install -d Ubuntu 22.04.01
+        ```
+        You will see something like this:
+        Installing, this may take a few minutes...
+        Please create a default UNIX user account. The username does not need to match your Windows username.
+        For more information visit: https://aka.ms/wslusers
+        Enter new UNIX username:
+
+        ## 2.3 Enter your new username and password at the prompts, and you will eventually see something like this
+        ```
+        Welcome to Ubuntu 22.04.1 LTS (GNU/Linux 5.10.102.1-microsoft-standard-WSL2 x86_64)
+        ```
+
+        # 2.4 Click on the Windows "Start" icon and type in the name of your distribution into the search box.
+        Your new linux distribution should appear as a local "App". You can pin it to the Windows task bar or to Start for your future convenience.
+        Start a VM using your distribution by clicking on the App.
+
+        # 3. Install pre-requisites into your new VM
+        ## 3.1 update apt-get packages
+        ```
+        sudo apt-get update
+        ```
+        ## 3.2 Install golang
+        ```
+        wget https://golang.org/dl/go1.19.linux-amd64.tar.gz
+        sudo tar -zxvf go1.19.linux-amd64.tar.gz -C /usr/local
+        echo export GOROOT=/usr/local/go | sudo tee -a /etc/profile
+        echo export PATH=$PATH:/usr/local/go/bin | sudo tee -a /etc/profile
+        source /etc/profile
+        go version
+        ```
+        ######### the following didn't work, probably because the kcp wasn't installed yet - no instruction re kcp install
+        go install ./cmd/kcp ./cmd/kubectl-*
+        #######################################################
+        ```
+        ## 3.3 Install ko (but don't do ko set action step)
+        ```
+        go install github.com/google/ko@latest
+        ```
+
+        ## 3.4 Install gcc
+        You should be able to "sudo apt install build-essential" theoretically, but I had an issue with it
+        ```
+        sudo apt-get update
+        apt install gcc
+        gcc --version
+        ```
+        ## 3.5 Install make
+        ```
+        apt install make
+        ```
+        ## 3.6 Install jq
+        ```
+        DEBIAN_FRONTEND=noninteractive apt-get install -y jq
+        jq --version
+        ```
+
+        ## 3.6 Install docker
+        The installation instructions from docker are not sufficient to get docker working with WSL
+
+        ## 3.6.1 
+        Follow instructions here to install docker https://docs.docker.com/engine/install/ubuntu/
+
+        Here some additonal steps you will need to take:
+
+        ## 3.6.2 Edit /etc/wsl.conf so that systemd will run on booting:
+        ```
+        vi /etc/wsl.conf
+        ```
+        Insert
+        ```
+        [boot]
+        systemd=true
+        ```
+        ## 3.6.3 Edit /etc/sudoers 
+        ```
+        vi /etc/sudoers
```

Comment:
it is strongly recommended to not add directives directly to /etc/sudoers, but instead to put them in files in /etc/sudoers.d which are auto-included. So instead, make/modify a new file via
`sudo vi /etc/sudoers.d/docker`

Reply:
Hi @KPRoche thank you for your reviews! Where did you find the information regarding best practices for /etc/sudoers? Is that just common knowedge?

Reply:
It's actually **in** the /etc/sudoers file if your ubuntu distro supports it.
```
#
# This file MUST be edited with the 'visudo' command as root.
#
# Please consider adding local content in /etc/sudoers.d/ instead of
# directly modifying this file.
#
# See the man page for details on how to write a sudoers file.
#

```

Reply:
Thanks!